### ISSUE TITLE
isparta now unmaintained... switch to babel-istanbul?

### ISSUE THREAD

**Issue by tribou** [nkbt/react-component-template#189]
I appreciate the example you've given here as it's helped me in writing my own React tests and producing code coverage.

I noticed that the [`isparta`](https://github.com/douglasduteil/isparta) package now has the "No Maintenance Intended" banner, and it looks like [`babel-istanbul`](https://github.com/jmcriffey/babel-istanbul) might now be more active.  After testing the switch on my own project, the coverage result appears to be the same.  You may just need to change this command in a couple places in your `package-scripts.js`:

``` sh
npm install --save babel-istanbul

# then change from
babel-node node_modules/isparta/bin/isparta cover

# to
babel-istanbul cover
```

**Comment by nkbt** [comment]
Totally right. Needs switching over. Keen to PR?

Thank you!

**Comment by nkbt** [comment]
Fixed by #190
